gallery/plot_repurposing_annotations.py

```diff
@@ -69,5 +38,31 @@ def plot(imgs, with_orig=True, row_title=None, **imshow_kwargs):
 # ~~~~~~~~~~~~~~~~~~~~~~~
 # For example, the masks to bounding_boxes operation can be used to transform masks into bounding boxes that can be
 # used in methods like Faster RCNN and YOLO.
-padded_imgs = [T.Pad(padding=padding)(orig_img) for padding in (3, 10, 30, 50)]
-plot(padded_imgs)
+
+image, labels = skimage.draw.random_shapes((512, 256), 4, min_size=32, multichannel=False)
+
+labeled_image = skimage.measure.label(image, background=255)
+
+labeled_image = skimage.img_as_ubyte(labeled_image)
```

Comment:
Could we avoid introducing a `skimage` dependency on the docs? It seems we could do something like this using pytorch.

Reply:
I'm using it for `random_shapes`. I suppose I could use the text fixture images.